src/hal/link/ping-port.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <stdint.h>
+
+// port interface used by PingDevice class
+class PingPort {
+public:
+    virtual int read(uint8_t* buffer, int nBytes)      = 0;
```

Comment:
weird whitespace

Reply:
From the original #9 patch, I though that I have used clang-format, will run it inside this folder to make sure.

Reply:
clang-format used.